src/Tests/dotnet-new.Tests/DotnetClassTemplateTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Extensions.Logging;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.TemplateEngine.Authoring.TemplateVerifier;
+using Microsoft.TemplateEngine.TestHelper;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Cli.New.IntegrationTests
+{
+    public class DotnetClassTemplateTests : BaseIntegrationTest, IClassFixture<SharedHomeDirectory>
+    {
+        private readonly SharedHomeDirectory _fixture;
+        private readonly ITestOutputHelper _log;
+        private readonly ILogger _logger;
+
+        public DotnetClassTemplateTests(SharedHomeDirectory fixture, ITestOutputHelper log) : base(log)
+        {
+            _fixture = fixture;
+            _log = log;
+            _logger = new XunitLoggerProvider(log)
+                .CreateLogger("TestRun");
+        }
+
+        [Theory]
+        [InlineData("class")]
+        [InlineData("class", "9.0", "netstandard2.0")]
+        [InlineData("interface")]
+        [InlineData("interface", "9.0", "netstandard2.0")]
+        [InlineData("record")]
+        [InlineData("record", "9.0")]
+        [InlineData("record", "8.0", "netstandard2.0")]
+        [InlineData("struct")]
+        [InlineData("struct", "9.0", "netstandard2.0")]
+        [InlineData("enum")]
+        [InlineData("enum", "9.0", "netstandard2.0")]
+        public async void DotnetCSharpClassTemplatesTest(
+            string templateShortName,
+            string langVersion = "preview",
```

Comment:
This will cause the `LangVersion` always be present in csproj. Please consider test cases also when it is not specified in csproj.